### PR TITLE
add Stringer interface for eventType

### PR DESCRIPTION
**Pull request by weiwei04**
**What this PR does / why we need it**:

fix invalid log outputs like 

"graph_builder.go:429] GraphBuilder process object: v1/Endpoints, namespace kube-system, name kube-controller-manager, event type %!s(garbagecollector.eventType=1)"

### CODE REVIEW

**Comment by weiwei04** [comment]
I signed it

File: pkg/controller/garbagecollector/graph_builder.go [hunk 1]
```diff
@@ -427,7 +438,7 @@ func (gb *GraphBuilder) processGraphChanges() bool {
 		utilruntime.HandleError(fmt.Errorf("cannot access obj: %v", err))
 		return true
 	}
-	glog.V(5).Infof("GraphBuilder process object: %s/%s, namespace %s, name %s, event type %s", typeAccessor.GetAPIVersion(), typeAccessor.GetKind(), accessor.GetNamespace(), accessor.GetName(), event.eventType)
```
Comment (deads2k) [inline pkg/controller/garbagecollector/graph_builder.go]: With a stringer interface, can't you use `%v` and skip adding the `.String()`?

Comment (weiwei04) [inline pkg/controller/garbagecollector/graph_builder.go]: that would be better

**Comment by deads2k** [comment]
lgtm, squash for merge

**Comment by weiwei04** [comment]
@deads2k squashed, ptal.

**Pull request merged**